### ISSUE TITLE
CLI failing with SyntaxError: Unexpected token '.' and node 12.21.0

### ISSUE THREAD

**Issue by Fabian-K** [SAP/ui5-tooling#663]
Hi UI5 Team,

versions >= 2.14.11 of the ui5 cli seem to be broken when using node 12.21.0

Thanks, Fabian

## Expected Behavior
`ui5 --help` working

## Current Behavior
`ui5 --help` fails 

## Steps to Reproduce the Issue

```
FROM node:12.21.0
RUN npm install --global @ui5/cli@2.14.13
RUN ui5 --help
```
## Context
 * Node.js Version: **`v12.21.0`**
 * npm Version: `6.14.11`
 * last working version of @ui5/cli seems to be 2.14.10

## Log Output / Stack Trace

```
ui5 --help

(node:7) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token '.'
    at Loader.moduleStrategy (internal/modules/esm/translators.js:140:18)
(node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by RandomByte** [comment]
Thanks for reporting! This seems to be related to https://github.com/SAP/ui5-cli/pull/533

I was able to reproduce this locally with Node `v12.21.0` as well as `v12.22.12` and npm `6.14.16´ as well as npm 8.

Our CI is using `v12.22.12` but unfortunately did not catch this.

**Comment by RandomByte** [comment]
update-notifier in the latest version `6.0.2`, which we use, [does not seem to support Node 12](https://github.com/yeoman/update-notifier/blob/3046d0f61a57f8270291b6ab271f8a12df8421a6/package.json#L16).

I think our CI could not catch this, since we intentionally disable the update-notifier in CI environments: https://github.com/SAP/ui5-cli/blob/12416982efcc9d76cc78832c472516d8eaf05185/bin/ui5.js#L66

**Comment by matz3** [comment]
Fixed with [@ui5/cli v2.14.14](https://github.com/SAP/ui5-cli/releases/tag/v2.14.14)
